plugins/lua/buildingplan/planneroverlay.lua

```diff
@@ -422,17 +434,28 @@ function PlannerOverlay:init()
         },
         widgets.CycleHotkeyLabel {
             view_id='weapons',
-            frame={t=5, l=4},
+            frame={b=4, l=1, w=22},
             key='CUSTOM_T',
             key_back='CUSTOM_SHIFT_T',
-            label='Num weapons:',
+            label='# of weapons:',
```

Comment:
I think "Number of weapons" is more readable -- we have the space for it anyway.

Reply:
no we don't! 😭 
![bplanoverlay_6_weaptrap3](https://user-images.githubusercontent.com/5853667/229662569-d940a630-6f58-4a25-a19c-eba4e6450970.png)

Reply:
I used the longer text for now. We can re-shorten once we add the slider